### DISCUSSION TITLE
Mountain Duck-4.16.0.22153 won't open on MacOS Sonoma 14.4.1 (23E224)

### DISCUSSION THREAD

**Discussion by mvosz** [iterate-ch/cyberduck#15993] [Q&A] [answered]
I've updated Mountain Duck today and I cannot see the Mountain Duck Icon in the Menu Bar. I've restartet my Mac, but still cannot see Mountain Duck. But it mounted a webdav for me, which is something.

**Comment by tatze96** [comment, marked as answer]
There are two possible explanations if the duck symbol doesn't appear within the Status Bar area.
1. You are using applications that hide the icon. This was previously reported for "xbar", "menu meters", and "IP in menubar".
2. You have many status bar items open so that they reach the middle of the screen or, more specifically, the notch area. In that case, macOS will hide any items beyond the notch. There is no workaround for this behavior other than removing currently not required icons.

Please check if either of the aforementioned options is the case.
